components/layout/grid/src/gridItem/index.js

```diff
@@ -1,10 +1,30 @@
 import PropTypes from 'prop-types'
 import cx from 'classnames'
 
-import {BASE_CLASS, CELL_NUMBERS} from '../settings'
+import {BASE_CLASS, CELL_NUMBERS, BREAKPOINTS} from '../settings'
+
+const getColSpanClassNamesTransform = ({colSpan, ...otherProps}) => {
+  const className = []
+  if (CELL_NUMBERS.includes(colSpan) && otherProps.xxs === undefined) {
+    className.push(`${BASE_CLASS}-item--xxs-${colSpan}`)
```

Comment:
as this is only one key and nothing else will be added, you could just return the string and avoid the push/join cost

Reply:
done in the reduce